pkg/exec/helpers.go

```diff
@@ -132,3 +133,23 @@ func RunWithStdinWriter(cmd Cmd, writerFunc func(io.Writer) error) error {
 		},
 	})
 }
+
+// ToPath is a helper to run a cmd and write the output to path
+func ToPath(cmd Cmd, path string) error {
```

Comment:
this probably needs a more descriptive title, `exec.ToPath` isn't a very obvious function out of context.

imo this should be an internal helper of the provider log collection